redshift/redshift.go

```diff
@@ -19,59 +17,55 @@ type dbExecCloser interface {
 	Exec(query string, args ...interface{}) (sql.Result, error)
 }
 
+// S3Info holds the information necessary to copy data from s3 buckets
+type S3Info struct {
+	Region    string
```

Comment:
The region is a property of the s3 bucket being read from/copied to and not the redshift cluster. In the sense that we need to set `aws_region` params if the s3 bucket being used is in a different region than the redshift cluster. The value for this variable might be different based on the bucket you are interfacing with.